source/Calamari/Kubernetes/Conventions/GatherAndApplyRawYamlConvention.cs

```diff
@@ -135,19 +129,15 @@ private IEnumerable<Resource> ApplyBatchAndReturnResources(int index, string glo
                 var token = JToken.Parse(outputJson);
 
                 List<Resource> lastResources;
-                if (token.Type == JTokenType.Array)
-                {
-                    lastResources = token.ToObject<List<Resource>>();
-                }
```

Comment:
This check was removed because I thought it was required at first when I added it but after more testing it seems like I was just mistaken when porting the logic from bash/powershell to c# so I've removed it to reduce confusion for future engineers.